postgres/messaging/postgresql.conf

```diff
@@ -25,3 +25,5 @@ shared_preload_libraries = 'pg_stat_statements,pg_cron,pg_partman_bgw'
 pg_partman_bgw.dbname = 'postgres'
 pg_partman_bgw.interval = 60
 pg_partman_bgw.role = 'postgres'
+
+include_if_exists 'tembo.postgresql.conf'
```

Comment:
per our conversation, change this to be `include_if_exists '/tembo/config/tembo.postgresql.conf`